star.cpp

```diff
@@ -123,15 +123,16 @@ static double hfr(std::vector<R2M>& vec, double cx, double cy, double mass)
     return hfr;
 }
 
-bool Star::Find(const usImage *pImg, int searchRegion, int base_x, int base_y, FindMode mode, double minHFD, unsigned short maxADU)
+bool Star::Find(const usImage *pImg, int searchRegion, int base_x, int base_y, FindMode mode, double minHFD, unsigned short maxADU, bool minimalLogging)

```

Comment:
I like the idea of having optional verbose logging, but bool function params can be pretty opaque and the reader of the calling code won't readily understand the meaning of the true/false being passed.

An equivalent but more readable option would be to create an `enum` either global or in `class Star` so that the callers can pass a named value:
```
enum StarFindLogType {
    FIND_LOGGING_MINIMAL,
    FIND_LOGGING_VERBOSE,
};
```
then callers pass one of those values and the function signature would be:
 ```suggestion
bool Star::Find(const usImage *pImg, int searchRegion, int base_x, int base_y, FindMode mode, double minHFD, unsigned short maxADU, StarFindLogType logging)
```